NetKet/GroundState/ground_state.hpp

```diff
@@ -49,8 +49,8 @@ class GroundState {
       throw InvalidInputError(s.str());
     }
 
-    Graph graph(pars);
-    Hilbert hilbert(graph, pars);
+    auto graph = make_graph(pars);
+    Hilbert hilbert(*graph, pars);
```

Comment:
No, because pointers have a nothing state while `make_hilbert` most likely assumes that we're passing it a valid graph. `*` can be avoided using references like I've shown.